README.md

```diff
@@ -80,7 +73,19 @@ assert!(text_result.as_text().starts_with("; SPIR-V\n"));
 Setup
 -----
 
-To build the shaderc-rs crate, the following tools must be installed and available on `PATH`:
+Building the underlying libshaderc from source takes a while.  You can provide it or use system libraries by setting the `SHADERC_STATIC` path.  On Linux, the build script also checks for independent libglslang and libSPIRV and includes them only if found.
```

Comment:
I think the explanation here is not very clear. It would be nice to document features, env vars, and lib resolution orders more clearly, like:

* The default behavior is to search and link system libraries for shaderc under the directory given by env var .... This first tries static library, then dynamic library. If none is found, then ...
* If `--no-default-features` is given, then ... This is deprecated ...
* If `build-from-source` is given, then ...